public/locales/en.json

```diff
@@ -446,5 +446,51 @@
     "adminForEvents": "Admin for events",
     "addedAsAdmin": "User is added as admin.",
     "talawaApiUnavailable": "Talawa-API service is unavailable. Is it running? Check your network connectivity too."
+  },
+  "userNavbar": {
+    "talawa_portal": "Talawa",
+    "home": "Home",
+    "people": "People",
+    "events": "Events",
+    "chat": "Chat",
+    "donate": "Donate",
+    "myTasks": "My Tasks",
+    "settings": "Settings",
+    "language": "Language",
+    "logout": "Logout",
+    "close": "Close"
+  },
+  "userLogin": {
+    "login": "Login",
+    "forgotPassword": "Forgot Password?",
+    "loginIntoYourAccount": "Login into your account",
+    "emailAddress": "Email Address",
+    "enterEmail": "Enter your email address",
+    "password": "Password",
+    "enterPassword": "Enter your password",
+    "register": "Register",
+    "invalidDetailsMessage": "Please enter a valid email and password.",
+    "notAuthorised": "Sorry! you are not Authorised!",
+    "invalidCredentials": "Entered credentials are incorrect. Please enter valid credentials.",
```

Comment:
@beingnoble03 can you also add the link to the software/service used to get the translations. Just to know the credibility. Since there are lot of sentences. And when translating a sentence word by word the meaning and grammar might change in different different language. So we might need to validate this as we did in case of Tarawa mobile application.

Reply:
@CyberWake I used Google Translate to translate the sentences. I think it's pretty accurate.